<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1369</width>
    <height>770</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>670</y>
      <width>681</width>
      <height>36</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLineEdit" name="lint_send_data"/>
     </item>
     <item>
      <widget class="QPushButton" name="btn_send">
       <property name="text">
        <string>输入</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWebEngineView" name="wgt_map" native="true">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>70</y>
      <width>601</width>
      <height>621</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>211</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/picture/XH.png);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>271</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="hlt_view"/>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>681</width>
      <height>451</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>391</width>
       <height>421</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>10</y>
       <width>231</width>
       <height>301</height>
      </rect>
     </property>
     <property name="title">
      <string>无线数据传输</string>
     </property>
     <widget class="QComboBox" name="cmb_port_name">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>40</y>
        <width>99</width>
        <height>24</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>COM1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>COM2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>COM3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>COM4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>COM5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>COM6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>COM7</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>COM8</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="btn_close">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>230</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>close</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="chb_hex">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>105</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>HEX</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cmb_baud_rate">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>80</y>
        <width>99</width>
        <height>24</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>115200</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>57600</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>38400</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>19200</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>9600</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4800</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2400</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1200</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="btn_open">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>230</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>open</string>
      </property>
     </widget>
     <widget class="QLabel" name="lab_port">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>81</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>串口</string>
      </property>
     </widget>
     <widget class="QLabel" name="lab_sbaud">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>81</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>波特率</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="lab_status">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>180</y>
      <width>81</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1369</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">qwebengineview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
